packages/cli/src/helpers/flags.ts

```diff
@@ -177,3 +178,17 @@ export const urlOrFrame = flags.build({
   default: () => getConfigDefault('urlOrFrame'),
   description: 'URL or Frame',
 });
+
+export const emulateDevice = flags.build({
+  char: 'e',
+  default: () => getConfigDefault('emulateDevice', defaultFlagArgs.emulateDevice),
+  description: 'Simulate a device\'s screen size.',
+  options: deviceSettings.map(setting => setting.typeable),
+  parse: (s: string): EmulateDeviceSetting | undefined => {
```

Comment:
What is the use-case that we would want this to return `undefined`?

Reply:
Undefined would mean the user didn't select anything or the user selection doesn't exist. What's the best way to handle those cases?

Reply:
Ok I would remove `undefined` then. We want the ts to flag us if our implementation has anything that would allow the user to select a device that doesn't exist.